src/test/registration.test.js

```diff
@@ -38,61 +38,4 @@ describe('Tests for the registration of the users', () => {
         done(error);
       });
   });
-  it('Should not be possible to do any task if not an admin', (done) => {
```

Comment:
why did you delete these tests?

Reply:
Because we don't really need tests on readme and they were taking very long while this PR has to be the first to be merged for the others to be merged too.

Reply:
they are registration tests not about read me.
and they were their before